cyences_app_for_splunk/bin/cs_product_list.py

```diff
@@ -398,6 +402,14 @@ def build_app_dependency_search(app_list):
                 "search_values": "sophos:xg:firewall,sophos:xg:heartbeat,sophos:xg:system_health,sophos:xg:atp,sophos:xg:idp",
                 "earliest_time": "-1d@d",
                 "latest_time": "now",
+            },
+            {
+                "macro_name": "cs_sophos",
```

Comment:
Is the macro correct?

Reply:
Yes, we are collecting sophos_events data from sophos central addon in cs_sophos macro. & this data contains endpoint protection and firewall related logs.